diff -uraN cjs-4.8.0/cjs/context.cpp omv-cjs-4.8.0/cjs/context.cpp
--- cjs-4.8.0/cjs/context.cpp	2020-11-25 13:19:13.000000000 +0100
+++ omv-cjs-4.8.0/cjs/context.cpp	2020-11-27 20:59:47.727642531 +0100
@@ -293,7 +293,7 @@
     g_object_class_install_property(object_class, PROP_PROFILER_SIGUSR2, pspec);
     g_param_spec_unref(pspec);
 
-    /* For GjsPrivate */
+    /* For CjsPrivate */
     {
 #ifdef G_OS_WIN32
         extern HMODULE gjs_dll;
