diff -uraN cjs-4.6.0/cjs/context.cpp omv-cjs-4.6.0/cjs/context.cpp
--- cjs-4.6.0/cjs/context.cpp	2020-05-12 14:46:01.000000000 +0200
+++ omv-cjs-4.6.0/cjs/context.cpp	2020-05-26 19:52:44.542271578 +0200
@@ -283,7 +283,7 @@
     g_object_class_install_property(object_class, PROP_PROFILER_SIGUSR2, pspec);
     g_param_spec_unref(pspec);
 
-    /* For GjsPrivate */
+    /* For CjsPrivate */
     {
 #ifdef G_OS_WIN32
         extern HMODULE gjs_dll;
@@ -1091,4 +1091,4 @@
 gjs_get_js_version(void)
 {
     return JS_GetImplementationVersion();
-}
\ No newline at end of file
+}
